spec/omniauth/strategies/pam_spec.rb

```diff
@@ -63,17 +76,12 @@ def invalid_credentials
   end
 
   def mock_rpam(username:, password:, opts:)
-    allow(Rpam).to receive(:auth).with(username, password, opts)
+    allow(Rpam2).to receive(:auth).with(opts[:service], username, password)
```

Comment:
If we're using a fake with `Rpam2` now, I wonder if we need the mocking too. What do you think?

Reply:
sry for the late answer: no we don't need it anymore.